modules/gmc-agent/src/main/java/org/gridgain/action/controller/BaselineActionsController.java

```diff
@@ -50,12 +56,26 @@ public BaselineActionsController(GridKernalContext ctx) {
     }
 
     /**
-     * @param isAutoAdjustEnabled Is auto adjust enabled.
+     * @param awaitingTime Awaiting time in ms.
```

Comment:
Missing `@return`. Here and in many other places.

Reply:
Fixed